### PR TITLE
Add first draft of newsletter

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: docs/newsletters/newsletter-2023-06-15.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
+---
+layout: doc
+title: Inaugural issue of the OBO Foundry Newsletter
```
Comment (cthoyt) [inline docs/newsletters/newsletter-2023-06-15.md:3]: date of publication?

**Comment by cthoyt** [comment]
I would suggest using the built in blogging functionality of jekyll for this and instead put this in to https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/_posts

**Comment by matentzn** [comment]
Oh cool, yes - I just move it over and change the header block? Will do now.

File: _posts/2023-06-16-InauguralNewsletter.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
-title: Inaugural issue of the OBO Foundry Newsletter
+layout: post
+title: "OBO Inaugural Newsletter June 2023"
+date: 2023-06-16
+categories: update
+summary: "The inaugural edition of our newly formed OBO Newsletter."
 ---
 
-## Welcome to the inaugural issue of the OBO Foundry Newsletter -  Celebrating our 20th anniversary
+## Welcome to the inaugural issue of the OBO Foundry Newsletter -  Celebrating our 20th anniversary (June 2023)
```
Comment (cthoyt) [inline _posts/2023-06-16-InauguralNewsletter.md:9]: put this whole header into the title and remove the title. Then the first paragraph becomes the short text displayed on the blog roll

File: docs/newsletters/newsletter-2023-06-15.md (hunk 1 above)
Comment (matentzn) [reply docs/newsletters/newsletter-2023-06-15.md]: Done.

File: _posts/2023-06-16-InauguralNewsletter.md (hunk 2 above)
Comment (matentzn) [reply _posts/2023-06-16-InauguralNewsletter.md]: Ok done, thank you!

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**